recipes/spikingjelly/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "spikingjelly" %}
+{% set version = "0.0.0.0.8" %}
```

Comment:
That is an odd version number :smile:

Reply:
I thought so too - they must have big plans ;)